prov/verbs/src/verbs_info.c

```diff
@@ -361,7 +362,12 @@ int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 		return -FI_ENODATA;
 	}
 
-	if (attr->total_buffered_recv > info->rx_attr->total_buffered_recv) {
+	rm_enabled =(info->domain_attr &&
+		     info->domain_attr->resource_mgmt == FI_RM_ENABLED);
+
+	if (!rm_enabled &&
+	    (attr->total_buffered_recv > info->rx_attr->total_buffered_recv))
+	{
```

Comment:
Okay, I think I'm understanding your point. It seems the man page statement could be interpreted as total_buffered_recv value being meaningless when domain supports FI_RM_ENABLED since the actual buffering provided might be higher.

I thought total_buffered_recv was the minimum value of the buffering supported by the provider and if the app requests something higher, it has to be rejected. Now I'm getting doubts as to whether this is an input parameter (app requesting the provider to allocate required buffering) or output parameter (provider specifying the amount of buffering supported). 

@shefty Can you clarify the man page description? I hope you notice the email notification for this comment in the mountain of emails that you're sifting through. :)